mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -307,7 +314,14 @@ def _extract_headers_from_credentials(self, headers):
         return {header.name: header.value for header in headers}
 
     def _azure_upload_chunk(
-        self, credentials, headers, local_file, artifact_file_path, start_byte, size
+        self,
+        credentials,
+        headers,
+        local_file,
+        artifact_file_path,
+        start_byte,
+        size,
+        get_credentials=None,
```

Comment:
Can we add docstrings for `get_credentials` to every API where we're adding this param?

Reply:
got it :)